tests/test-completer/src/model.spec.ts

```diff
@@ -3,13 +3,17 @@
 
 import { expect } from 'chai';
 
-import { toArray } from '@lumino/algorithm';
+// import { toArray } from '@lumino/algorithm';
 
-import { JSONExt } from '@lumino/coreutils';
+// import { JSONExt } from '@lumino/coreutils';
```

Comment:
Should delete? Also, tests seem to be failing because this dependency is no longer used